xlson/app.py

```diff
@@ -0,0 +1,42 @@
+""" Importing required strings """
+import os
+
+from django import forms
+from django.views.generic.edit import FormView
+from django_micro import configure, run
+
+DEBUG = True
+configure(locals())
+
+
+CONTENT_TYPES = [".xlsx", ".xls"]
+MAX_UPLOAD_SIZE = 10485760
+
+
+class UploadFile(forms.Form):
+    """field in the upload form"""
+
+    files = forms.FileField()
+
+    def clean_files(self) -> str:
+        """clean data recieved from the upload form """
+        files = self.cleaned_data.get("files")
+        ext = os.path.splitext(files.name)[-1]
+        file_size = files.size
+        if ext in CONTENT_TYPES:
```

Comment:
We need to access these settings in the standard way that one gets things from django settings e.g.

```python
from django.conf import settings
content_types = settings.CONTENT_TYPES
```